habitat/default.toml

```diff
@@ -1,11 +1,28 @@
-# The Chef run interval in seconds
-interval = 10 
+[chef-client]
```

Comment:
the more I think about it - I don't really like having that chef-client table there at all. I think I'll remove it unless you have any objections. After looking through the docs there are only a couple of nested keys so it's sort of redundant to have a chef-client table in the chef-client package.